src/nxdoc/getting-started/tracks/package-publish-deploy-hotreload.md

```diff
@@ -0,0 +1,130 @@
+---
+title: 4 - Package \ Publish \ Deploy \ Hotreload
+review:
+    comment: ''
+    date: '2021-03-02'
+    status: ok
+details:
+    howto:
+        excerpt: Learn how to package, publish, deploy and hotreload a project.
+        level: Intermediate
+        tool: 'Nuxeo CLI, Nuxeo Studio'
+        topics: 'Automation, Layout'
+labels:
+    - howto
+    - nuxeo-cli
+toc: true
+tree_item_index: 300
+history:
+    -
+        author: Florian Bématol
+        date: '2021-03-02 15:26'
+        message: ''
+        version: '1'  
+---
+
+{{#> callout type='info'}}
+Watch the related courses on Nuxeo University
+- [Course on Nuxeo CLI](https://university.nuxeo.com/learn/public/course/view/elearning/83/NuxeoDevTools)
+{{/callout}}
+
+## Bootstrap your nuxeo package
+
+1.  In your terminal:
+
+    ```bash
+    $ nuxeo bootstrap package
+    ```
+
+2.  Link your project:
+
+    ```bash
+    $ nuxeo studio link
+    ```
+
+## Create project your docker image
+
+1.  In your terminal:
+
+    ```bash
+    $ nuxeo bootstrap docker-compose
+    ```
+
+## Run your project
+
+1.  In your terminal:
+
+    ```bash
+    $ docker-compose up [-d]
+    ```
+
+2.  If you want to follow the logs:
+
+    ```bash
+    $ docker-compose logs -f
+    ```
+
+#### Hotreload
+
+## Hotreload - Navigateur
+
+1.  Install extension
+
+2.  On your browser open your nuxeo application
+
+3.  Click extension logo, click hotreload button.
```

Comment:
I believe there is a doc page for the extension (not sure though), maybe link to it directly, it may contain nice screenshots